.gitlab-ci.yml

```diff
@@ -37,6 +37,18 @@ dunfell-raspberrypi3-mesa-weston-wpe-2.30:
     - bitbake $BITBAKE_TARGET
     - rm -rf tmp
 
+dunfell-raspberrypi3-mesa-weston-wpe-2.32:
```

Comment:
Please also remove the `dunfell-raspberrypi3-mesa-weston-wpe-2.30` task.

Reply:
My bad, I noticed 2.30 is still there so it looks good to keep the job for build that recipe too.

Reply:
Testing the builds in https://gitlab.com/browsers/meta-webkit/-/pipelines/367127981